manifests/machineconfigserver/clusterrole.yaml

```diff
@@ -13,3 +13,6 @@ rules:
   resourceNames: ["hostnetwork"]
   resources: ["securitycontextconstraints"]
   verbs: ["use"]
+- apiGroups: [""]
```

Comment:
reminder to remove this and see if it still works

Reply:
the only configmap we're getting is:

```
cm, cmErr = dn.kubeClient.CoreV1().ConfigMaps("openshift-machine-config-operator").Get(context.TODO(), "kubeconfig-data", v1.GetOptions{})
```

in the daemon right now right? We have the role above to cover that no? Everything else is in the operator, and we don't have an informer anymore

Reply:
Sorry, I thought this was the daemon :facepalm: ignore me